webpack.common.js

```diff
@@ -65,6 +65,14 @@ module.exports = {
 			template: 'src/index.html',
 			favicon: './src/styles/images/favicon.ico'
 		}),
+		// Global constants, which should mirror the global constants in jest.config.js
+		new webpack.DefinePlugin({
+			CONCH: {
+				GLOBALS: {
+					apiUrl: JSON.stringify("http://localhost:5001"),
```

Comment:
This I think needs different values for production/staging vs. dev so I think we need to push it down into the two lower webpack files maybe?

Reply:
@perigrin I think the latest changes (making the use of the api url optional) makes it so that we can leave this in the common config. Let me know if you disagree or if you have any additional thoughts or recommendations on this.